dropwizard-client/src/test/java/io/dropwizard/client/ConfiguredCloseableHttpClientTest.java

```diff
@@ -0,0 +1,137 @@
+package io.dropwizard.client;
+
+import org.apache.http.HttpHost;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.conn.ClientConnectionManager;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.params.HttpParams;
+import org.apache.http.protocol.HttpContext;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+
+import static org.hamcrest.core.Is.is;
```

Comment:
Please, use [AssertJ](http://joel-costigliola.github.io/assertj/) for consistency with the rest of the codebase.